B827EBFFFE28DC7F.json

```diff
@@ -0,0 +1,18 @@
+{
++  "gateway_conf": {
++    "gateway_ID": "B827EBFFFE28DC7F",
++    "servers": [
```

Comment:
Looks like you copy&pasted another diff file, because the content ended up full of those `+` signs before each line, making it an invalid json file. Please fix it and then I'll merge. Thx!